.github/workflows/integration_bugtracker.yml

```diff
@@ -48,6 +48,8 @@ jobs:
         export GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN="${{ secrets.GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_USERNAME="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_USERNAME }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_TOKEN="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_TOKEN }}"
+        export BITBUCKET_INTEGRATION_API_USERNAME="${{ secrets.BITBUCKET_INTEGRATION_API_USERNAME }}"
```

Comment:
If we are using the "App password" then username won't be needed. Let's do that (will need to be documented how the user can generate an app password).

Reply:
Username is needed different from the other "access token" mechanisms. See [Comment](https://github.com/kiwitcms/Kiwi/pull/2053#discussion_r517976264)